source/service.js

```diff
@@ -1,26 +1,33 @@
-import { getJSON, global } from './dom';
+import { global } from './dom';
 
-/**
- * @param {Function} updateBroadcaster
- */
-const defaultFetch = function (updateBroadcaster) {
-    getJSON(updateBroadcaster.url, (count) => {
-        try {
-            const convertedNumber = typeof this.convertNumber === 'function' ? this.convertNumber(count) : count;
+function fetchXHR(updateBroadcaster) {
```

Comment:
XHR/JSON is not a common default. 
JSONP can be still used on case-by-case basis, the only one with it in 2020 is VK